
[34m[1mwandb[39m[22m: [33mWARNING[39m No relevant files were detected in the specified directory. No code will be logged to your run.
Experiment dir: ./exp/Test_ihm_south_baseline
['/data/datasets/mimic3-benchmarks/data/in-hospital-mortality', '/data/datasets/eICU2MIMIC/ihm_split']
Traceback (most recent call last):
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/scripts/../tests/test.py", line 614, in <module>
    main()
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/scripts/../tests/test.py", line 373, in main
    TestLSTM().test_standard_lstm(
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/tests/test_standard_lstm.py", line 330, in test_standard_lstm
    test_loaders = get_test_loaders(
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/utils.py", line 222, in get_test_loaders
    test_dataset = IHMDataset(
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/ihm/dataset.py", line 59, in __init__
    self._read_data(root, listfile)
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/ihm/dataset.py", line 101, in _read_data
    self.reader.read_example(0)["X"]
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/readers.py", line 177, in read_example
    (X, header) = self._read_timeseries(name)
  File "/home/grads/k/kingrc15/Continual_Learning/Continual_Learning/torchmimic/data/readers.py", line 130, in _read_timeseries
    with open(os.path.join(self._dataset_dir, ts_filename), "r") as tsfile:
FileNotFoundError: [Errno 2] No such file or directory: '/data/datasets/mimic3-benchmarks/data/in-hospital-mortality/test/10011_episode232110_timeseries_intime_2126-08-14_22-34-00_outtime_2126-08-28_18-59-00.csv'